tests/foreman/cli/test_ping.py

```diff
@@ -56,3 +56,27 @@ def test_positive_ping():
         assert result.return_code == 0, 'Return code should be 0 if all services are ok'
     else:
         assert result.return_code != 0, 'Return code should not be 0 if any service is not ok'
+
+
+@pytest.mark.tier1
+def test_negative_ping_fail_status_code():
+    """Negative test to verify non-zero status code of ping fail
+
+    :id: 8f8675aa-df52-11eb-9353-b0a460e02491
+
+    :BZ: 1941240
+
+    :expectedresults: Hammer ping fails and returns non-zero(1) status code.
+
```

Comment:
Please add also `:CustomerScenario: true`, it's because 1941240 has got a customer case attached

Reply:
Added